ansible/roles/test/tasks/pfc_wd/functional_test/storm_all_test.yml

```diff
@@ -1,7 +1,7 @@
 - name: Prepare variables required for PFC test
   set_fact:
     pfc_queue_index: 4
-    pfc_frames_number: 1000000
+    pfc_frames_number: 10000000
```

Comment:
>10000000 [](start = 23, length = 8)

It's tricky to tune this number, and seems it is not a final solution. Please add documents, so let others tune it if something happens

Reply:
Done